### PR TITLE
Initial set up and Add Assessment form

### PR DESCRIPTION
**Pull request by Danwhy**
* adds initial set up for the app,
* adds routing,
* adds form for students adding assessment

ref #2

### CODE REVIEW

**Review by iteles — CHANGES_REQUESTED** [review]
@Danwhy Could we please add a bit more to the readme so we have an explanation of what's happening and why it's in a separate repo with our first merge?

**Review by iteles — APPROVED** [review]
Good enough so we don't just have a blank readme 👍 Approving to remove my 'request changes' from before

**Comment by Danwhy** [comment]
:+1: I'll update it as more features get added

**Comment by sohilpandya** [comment]
Probably worth adding a `.nvmrc` file in with the version of node that we are using. So that we all have the same deps.

File: src/AddAssessment.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        />
+        <h2>Type of Assessment</h2>
+        <RadioGroup onChange={ selected => this.setState({type: selected}) } horizontal>
+          {["Exam", "Coursework"].map((v, i) => {
+            return <RadioButton value={v.toLowerCase()} key={i}>{v}</RadioButton>
+          })}
+        </RadioGroup>
+        <h2>Priority</h2>
+        <RadioGroup onChange={ selected => this.setState({priority: selected}) } horizontal>
+          {["Very Low", "Low", "Medium", "High", "Very High"].map((v, i) => {
```
Comment (sohilpandya) [inline src/AddAssessment.js]: Is this going to be a set list? Likely move this and line 45 out and create const for them and then use that in here. 

makes it slightly more readable.

**Review by sohilpandya — CHANGES_REQUESTED** [review]
Hey! looking good, just a couple of things on here.

**Comment by sohilpandya** [comment]
Also @Danwhy, I am unable to change labels etc in here. could you please add me to the team on here :)

**Review by sohilpandya — APPROVED** [review]

**Comment by sohilpandya** [comment]
LGTM 👍

**Pull request merged**